web/client/plugins/map/index.js

```diff
@@ -32,19 +32,21 @@ module.exports = (mapType, actions) => {
 
     const LMap = connect((state) => ({
         projectionDefs: projectionDefsSelector(state),
-        mousePosition: state.mousePosition || {enabled: false}
+        mousePosition: isMouseMoveActiveSelector(state),
+        isMouseMoveOverTools: get(state, 'map.present.mouseMoveOverTools', true)
```

Comment:
Not needed anymore, I think you can remove it

Reply:
Done